cmd/aks-periscope/aks-periscope.go

```diff
@@ -22,25 +23,33 @@ func main() {
 		log.Printf("Failed to create CRD: %+v", err)
 	}
 
+	clusterType := os.Getenv("CLUSTER_TYPE")
+
 	collectors := []interfaces.Collector{}
 	containerLogsCollector := collector.NewContainerLogsCollector(exporter)
 	collectors = append(collectors, containerLogsCollector)
-	systemLogsCollector := collector.NewSystemLogsCollector(exporter)
-	collectors = append(collectors, systemLogsCollector)
 	networkOutboundCollector := collector.NewNetworkOutboundCollector(5, exporter)
 	collectors = append(collectors, networkOutboundCollector)
-	ipTablesCollector := collector.NewIPTablesCollector(exporter)
-	collectors = append(collectors, ipTablesCollector)
-	nodeLogsCollector := collector.NewNodeLogsCollector(exporter)
-	collectors = append(collectors, nodeLogsCollector)
 	dnsCollector := collector.NewDNSCollector(exporter)
 	collectors = append(collectors, dnsCollector)
```

Comment:
This format means the whole block needs to be read to work out which collectors are enabled for a particular mode (connectedCluster vs default). 

Prefer to see all the "append" calls inside the if statement, even though it means more lines and some duplication, it makes it clearer exactly which collectors are enabled for each mode

Reply:
@davidkydd, just pushed new changes. If it looks better now, we can resolve this thread.

Reply:
perfect - thanks!